packages/grafana-data/src/field/fieldDisplay.ts

```diff
@@ -79,6 +79,7 @@ export interface GetFieldDisplayValuesOptions {
   replaceVariables: InterpolateFunction;
   sparkline?: boolean; // Calculate the sparkline
   theme: GrafanaTheme;
+  calculatePercent?: boolean;
```

Comment:
this is a flag to avoid calculating min/max when not necessary.  It will also be required when we add a 'relative' threshold mode in #20783

Reply:
Is there a reason you named this calculatePercent and not calculateMinMax or autoMinMax ?